packages/uui-dialog-layout/package.json

```diff
@@ -0,0 +1,43 @@
+{
+  "name": "@umbraco-ui/uui-dialog-layout",
+  "version": "0.0.0",
+  "license": "MIT",
+  "keywords": [
+    "Umbraco",
+    "Custom elements",
+    "Web components",
+    "UI",
+    "Lit",
+    "Dialog Layout"
+  ],
+  "description": "Umbraco UI dialog-layout component",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/umbraco/Umbraco.UI.git",
+    "directory": "packages/uui-dialog-layout"
+  },
+  "bugs": {
+    "url": "https://github.com/umbraco/Umbraco.UI/issues"
+  },
+  "main": "./dist/uui-dialog-layout.min.js",
+  "module": "./lib/index.js",
+  "customElements": "custom-elements.json",
+  "files": [
+    "dist",
```

Comment:
I think we shouldnt ship dist foler anymore. so this needs to go away.

@iOvergaard do we still create dist folder in files when we make a new package? or is this fixed in lastest?